### PR TITLE
Sprint 2- tests now run in a docker container

### PR DESCRIPTION
**Pull request by savatarko**
Integration tests now run in a docker container.

Market-service still not added into docker-compose test file, will add when I add tests for market-service.

Tests that use email service are currently disabled not to spam it, will think of a way to mock it using some kind of a api.

### CODE REVIEW

File: user-service/src/test/resources/application.properties.example [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
 spring.jpa.database-platform=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.generate-ddl=true
-spring.jpa.hibernate.ddl-auto=create
+spring.jpa.hibernate.ddl-auto=update
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.show-sql=true
 logging.level.org.hibernate.SQL=DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-server.port=8080
+server.port=8084
```
Comment (ijevtic) [inline user-service/src/test/resources/application.properties.example:13]: zasto ovo?

Comment (savatarko) [reply user-service/src/test/resources/application.properties.example:13]: ovo je samo kada se pokrecu testovi, ostalo mi je od ranije (nije pravi port)

Comment (ijevtic) [reply user-service/src/test/resources/application.properties.example:13]: skapirao sam, samo je random, da li ima neki odredjen razlog za to ili iz gasa

Comment (savatarko) [reply user-service/src/test/resources/application.properties.example:13]: iz gasa 🤫🧏🏻‍♂️

**Pull request merged**